docs/man/sat-bootsys.8.rst

```diff
@@ -17,57 +17,87 @@ SYNOPSIS
-safe to shutdown and power off ncn-w001 where this command was run.
+The shutdown action consists of the following stages. Each stage automates a
+specific portion of the system shutdown procedure.
+
+In the ``capture-state`` stage, ``sat bootsys`` saves the state of all
+Kubernetes pods to a file in an S3 bucket. This file can be used when the system
+is booted to verify all pods are in the states they were in before the shutdown.
+The S3 bucket must be configured in the ``s3`` section of the SAT configuration
+file.
+
+In the ``session-checks`` stage, it checks for any active sessions across
+multiple different services in the system, including the Boot Orchestration
+Service (BOS), the Configuration Framework Service (CFS), the Compute Rolling
+Upgrade Service (CRUS), the Firmware Action Service (FAS), the Node Memory Dump
+(NMD) service, and the System Dump Utility (SDU). If any active sessions are
+found, it will print information about those sessions and exit with exit code 1.
+If it does not find any active sessions, it will print messages to that effect
+and proceed with the next stage of the shutdown.
+
+In the ``bos-operations`` stage, it uses the Boot Orchestration Service (BOS) to
+shut down the compute nodes and User Access Nodes (UANs). It attempts to check
+whether the nodes are already in an "Off" state in the Hardware State Manager
+before attempting to shut them down with BOS.
+
+In the ``cabinet-power`` stage, it suspends the ``hms-discovery`` Kubernetes
+cron job and uses the Cray Advanced Platform Monitoring and Control (CAPMC)
+service to power off all liquid-cooled and air-cooled compute node cabinet and
+non-management NCN cabinets. It then waits for all components in those cabinets
+to reach the power "Off" state.
+
+In the ``platform-services`` stage, it stops all management services on the
+non-compute nodes (NCNs). This includes backing up and stopping the etcd
+cluster, stopping the kubelet service on each node, freezing ceph, and stopping
+all containers running in containerd. This ansible playbook is known to
+experience a problem that results in it hanging when stopping containers in
+containerd, so if this occurs, a workaround will automatically be executed that
+kills hanging processes on each node where it is necessary. Once the playbook
+has completed, it will move on to the next stage.
+
+In the ``ncn-power`` stage, it shuts down Linux on all NCNs, and powers them off
+using IPMI. Once this has completed, and it is safe to shutdown and power off
+ncn-m001 where this command was run.
 
 BOOT ACTION
 -----------
 
-The boot action is not currently implemented and will exit with a non-zero exit
-code and an error message.
+The boot action consists of the following stages. Similar to the shutdown
+action, each stage automates a specific portion of the system boot procedure.
+
+In the ``ncn-power`` stage, ``sat bootsys`` powers on all mangagement NCNs and
```

Comment:
Oh it also sets up screen sessions to monitor console logs using `ipmitool`. Might be helpful to mention that here and in the `ncn-power` stage of shutdown while we're at it.